x/oracle/keeper/common/expected_keepers.go

```diff
@@ -3,6 +3,7 @@ package common
 import (
 	"cosmossdk.io/math"
 	stakingkeeper "github.com/ExocoreNetwork/exocore/x/dogfood/keeper"
```

Comment:
can we unfied stakingkeeper  with dogfoodkeeper ?

Reply:
updated